pkg/services/payment_request/payment_request_reviewed_processor.go

```
@@ -80,13 +79,11 @@ func (p *paymentRequestReviewedProcessor) ProcessAndLockReviewedPR(pr models.Pay
 			SELECT * FROM payment_requests
 			WHERE id = $1 FOR UPDATE SKIP LOCKED;
 		`
-		err := p.db.RawQuery(query, pr.ID).First(&lockedPR)
-		if err != nil {
-			return fmt.Errorf("failure retrieving and locking payment request: %w", err)
-		}
+		p.db.RawQuery(query, pr.ID).First(&lockedPR)
```

I don't think we want to entirely ignore the error entirely here -- that can be a red flag to some of the static analysis tools (Ron had a big PR I reviewed recently to clear a lot of this up across the project).  Instead, we want to catch the `sql.ErrNoRows` and return on that, but do flag any other errors (like a database issue).